src/Microsoft.DotNet.Build.Tasks.Packaging/src/GetPackageVersion.cs

```diff
@@ -33,48 +33,15 @@ public override bool Execute()
                 Log.LogError("Files argument must be specified");
                 return false;
             }
-
-            // remove files that should be skipped or don't have a version
-            var filesToConsider = Files.Where(f => !f.GetMetadata("SkipVersionCheck").Equals("true", StringComparison.OrdinalIgnoreCase) &&
+            
+            var versionsToConsider = Files.Where(f => !f.GetMetadata("SkipVersionCheck").Equals("true", StringComparison.OrdinalIgnoreCase) &&
```

Comment:
I was keeping it because it was a way to opt out of participating in the version calculation, but I can remove it.  Looks like only one project was using it and I don't think that was doing anything.  I'll remove it and double check the nuspec diff.